### ISSUE TITLE
Entry has private access in LinkedHashMap

### ISSUE THREAD

**Issue by m-menon** [wymsee/cordova-imagePicker#54]
Thanks for the plugin. If I can fix this problem that I am encountering, this is gona be pretty useful to me. The error that I get is

src/com/synconset/ImageFetcher.java:[293,57] error: Entry has private access in LinkedHashMap

I am a newbie to java and I tried replacing 

<code>
    // Hard cache, with a fixed maximum capacity and a life duration
    private final HashMap Integer, Bitmap  sHardBitmapCache = new LinkedHashMap Integer, Bitmap(
</code>

with 

<code>
    // Hard cache, with a fixed maximum capacity and a life duration
    private final Map Integer, Bitmap sHardBitmapCache = new LinkedHashMap Integer, Bitmap(
</code>

AND

<code>
        @Override
        protected boolean removeEldestEntry(LinkedHashMap.Entry<Integer, Bitmap> eldest) {
            if (size() > HARD_CACHE_CAPACITY) {
                // Entries push-out of hard reference cache are transferred to
                // soft reference cache
</code>

with 

<code>
        @Override
        protected boolean removeEldestEntry(Map.Entry<Integer, Bitmap> eldest) {
            if (size() > HARD_CACHE_CAPACITY) {
                // Entries push-out of hard reference cache are transferred to
                // soft reference cache
</code>

In short I replaced "LinkedHashMap"  with "Map"

But when I do that I get the following error:

/Cloudlocater/src/com/synconset/ImageFetcher.java:[290,18] error: cannot find symbol
 class ImageFetcher
/Cloudlocater/src/com/synconset/ImageFetcher.java:[293,47] error: package Map does not exi

Any clue on how I could fix this?

Thanks once again,
M&M

**Comment by m-menon** [comment]
ok issue is sorted. I had forgotten this

import java.util.Map;

:)

Thanks for the plugin.

Cheers,
M&M
